schemas/create-labware/1-create-labware.avsc

```diff
@@ -54,7 +54,7 @@
                     "doc": "Unique id to identify the sample inside Sanger lims"
                   },
                   {
-                    "name": "location", "type": "string",
+                    "name": "location", "type": ["null", "int"], "default": null,
```

Comment:
Is this an int? Can't quite remember what this is for?

Reply:
Oh no!  Good catch.  I think I copied the nullability from elsewhere and didn't change the type!

Reply:
Actually @stevieing I wonder whether the change I made is the right one.  They've clearly not had problems with this and so they must always be sending a location.  Perhaps the correct thing to do here would be to remove the part of the `"doc"` here that says this can be optional and ensure the messages from the demo publisher provides this value.  I think that's how I noticed it wasn't correct.

The message parser in TOL lab share is not set up to accept an optional value for this string.  So it would fail validation even though the schema allows it (another reason not to duplicate the validation in TOL lab share).